US-CBP/cbp-theme
# chore: adding ci with travis to publish design system

#### What's this PR do?

- added travis ci config to publish the design system to github pages
- publish.sh script makes directories nested in current cbp-theme to publish ds artifacts
- minor change to ds-css config for webpack to exit script in production mode

#### Where should the reviewer start?

Take a look at the travis and publish.sh scripts/config

#### How should this be manually tested?
Run 
- npm run init-cbp-theme
- npm run init-cbp-ds 

- npm run build-cbp-theme
- npm run build-cbp-ds

Then

cp -rf cbp-theme/dist cbp-theme/kitchensink/


mkdir cbp-theme/kitchensink/design-system
mkdir -p cbp-theme/kitchensink/ds-css/dist
npm run copy-ds-to-cbp-theme

Go to kitchensink and run a local server and see if you can get to the original kitchen sink and the new /design-system

#### Any background context you want to provide? N/A
#### What are the relevant tickets? N/A
#### Screenshots (if appropriate) N/A
#### Questions:
- Is there a blog post? No
- Does the knowledge base need an update? No
